### PR TITLE
Updating bonus cell feature

### PR DESCRIPTION
**Pull request by lostintangent**
This PR simply updates the logic for bonus cells in the guestbook. It should be completely bug free and worthy of a quick scan + LGTM! 👍Let's get this merged and pushed straight to production 🔥

### CODE REVIEW

File: src/components/Grid/GridLegend.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 
 import styled from "styled-components";
 
-function GridLegend() {
+function GridLegend({ bonusLabel = "Bonus" }) {
```
Comment (arjun27) [inline src/components/Grid/GridLegend.js]: `bonusLabel` is not used anywhere. Can we remove it?

Comment (lostintangent) [reply src/components/Grid/GridLegend.js]: Hmm, good catch!

File: src/components/Grid/GridLegend.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from "react";
...
     return (
         <Container>
```
Comment (lostintangent) [inline src/components/Grid/GridLegend.js]: Test

File: src/model/signatureMatrix.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { isBonusCell, isSpecialCell } from "./bonusCells";
 import { signatures } from "./signatures";
+import { observable } from "mobx";
```
Comment (lostintangent) [inline src/model/signatureMatrix.js]:
```suggestion
import { observable } from "mobx";
```

File: src/model/signatureMatrix.js [hunk 4]
```diff
@@ -19,9 +20,10 @@ for (let x = 0; x < COLUMNS; x++) {
     const isBonus = isBonusCell(cellNumber);
     const isSpecial = isSpecialCell(cellNumber);
 
-    const signature = signatures.unshift();
-    matrix.push({ isBonus, isSpecial, signature });
+    const signature = signatures.shift();
```
Comment (lostintangent) [inline src/model/signatureMatrix.js]:
```suggestion
    const signature = signatures.unshift();
```

File: src/components/Grid/GridLegend.js (hunk 1 above)
Comment (lostintangent) [reply src/components/Grid/GridLegend.js]: Thanks again!

Comment (lostintangent) [reply src/components/Grid/GridLegend.js]: Wow!

**Review by saifulrizal — APPROVED** [review]
This appears to be a query regarding the appropriate content for an internet comment. The user is seeking guidance on what to write. The intent is to understand best practices for online communication. Therefore, the response should offer helpful suggestions for crafting an effective comment.

**Review by saifulrizal — APPROVED** [review]

**Pull request closed without merging**